src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -119,11 +119,13 @@
   </resheader>
   <data name="ShellBanner" xml:space="preserve">
     <value>Windows PowerShell 
-Copyright (C) Microsoft Corporation. All rights reserved.</value>
+Copyright (C) Microsoft Corporation. All rights reserved.
+Version: {0}</value>
   </data>
   <data name="ShellBannerNonWindowsPowerShell" xml:space="preserve">
     <value>PowerShell 
-Copyright (C) Microsoft Corporation. All rights reserved.</value>
+Copyright (C) Microsoft Corporation. All rights reserved.
+Version: {0}</value>
```

Comment:
I think the version proposed by @SteveL-MSFT originally reads nicer:

```
PowerShell {0}
Copyright (C) Microsoft Corporation. All rights reserved.
```

For example, here is how python banner looks like
```
Python 2.7.10 (default, Oct 18 2015, 00:31:39) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.72)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
```